addons/point_of_sale/static/src/js/models.js

```diff
@@ -2132,10 +2132,8 @@ exports.Order = Backbone.Model.extend({
             fiscal_position_id: this.fiscal_position ? this.fiscal_position.id : false
         };
     },
-    export_for_printing: function(){
+    _export_for_printing: function(){
```

Comment:
rename function to what it actually does, now there are 2 functions, public and private, with the same name: for instance `_get_receipt_for_export`

Reply:
@switch87 Done, thanks! force-pushed